polyply/src/load_library.py

```diff
@@ -71,12 +71,9 @@ def wrapper(parser, path, force_field):
         path = pathlib.Path(path)
         file_extension = path.suffix.casefold()[1:]
 
-        try:
+        if file_extension in FORCE_FIELD_PARSERS.keys():
             parser = FORCE_FIELD_PARSERS[file_extension]
             wrapper(parser, path, force_field)
```

Comment:
I think we might want to raise a LOGGER.warning when the file extension is not know.